x-pack/plugins/ml/common/types/capabilities.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+// eslint-disable-next-line @kbn/imports/no_boundary_crossing
 import { KibanaRequest } from '@kbn/core/server';
```

Comment:
If I understand the new rules right, this looks like it can be turned into `import type { KibanaRequest } from '@kbn/core/server';` so we can get rid of the comment?

Reply:
From my understanding, yes